run_tests.py

```diff
@@ -83,10 +83,22 @@ def run_all_tests(single_file, remote):
                 print("Skipping tests\n")
                 return_values.append(0)
                 continue
-    else:
+    elif single_file:
         return_values.append(run_tests_on_pages(
             os.path.join(reports_path, os.path.basename(single_file)), [single_file], os.path.dirname(single_file),
             top_issue_num, test_class=PageTests))
+    elif folder:
+        print("Running spelling tests on folder {}".format(folder))
+        files = os.listdir(folder)
+        files_to_test = []
+        for f in files:
```

Comment:
NITPICK - `f` is not a PEP8 compliant variable name and should be renamed to be longer and more descriptive to something like `file` or `file_item`